packages/rest/src/router/external-express-routes.ts

```diff
@@ -50,7 +50,10 @@ export class ExternalExpressRoutes {
     rootDir: string,
     options?: ServeStaticOptions,
   ) {
-    this._staticRoutes.use(path, express.static(rootDir, options));
+    this._staticRoutes.use(path, express.static(
+      rootDir,
+      options,
+    ) as Application);
```

Comment:
This is problematic. The `express.static()` returns a `RequestHandler`, which is NOT an instance of `Application`. The original compilation error might be caused by something else.